sdk/servicebus/service-bus/src/models.ts

```diff
@@ -23,6 +23,15 @@ export interface MessageHandlers<ReceivedMessageT> {
   processError(err: Error): Promise<void>;
 }
 
+/**
+ * @internal
+ * @ignore
+ */
+export interface InternalMessageHandlers<ReceivedMessageT>
+  extends MessageHandlers<ReceivedMessageT> {
+  processInitialize?: () => Promise<void>;
```

Comment:
Nit: Was this added to help with tests? If so, can we add in a comment to reflect the same?

Reply:
Comment added!